tools/auto-fuzz/templates/java-introspector/Dockerfile-template

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+##########################################################################
+FROM gcr.io/oss-fuzz-base/base-builder-jvm
+#RUN curl -L %s -o maven.zip && unzip maven.zip -d $SRC/maven && rm -rf maven.zip
+RUN curl -L %s -o jdk.tar.gz && tar zxf jdk.tar.gz && rm -rf jdk.tar.gz
+COPY maven.zip $SRC/maven.zip
+RUN unzip maven.zip -d $SRC/maven && rm ./maven.zip
+ENV JAVA_HOME="$SRC/%s"
+ENV PATH="$JAVA_HOME/bin:$SRC/maven/apache-maven-3.6.3/bin:$PATH"
+RUN git clone --depth 1 https://github.com/ossf/fuzz-introspector introspector
```

Comment:
Can we switch this so it uses the local version that exists for `auto-fuzz`? THat makes it easier to develop I assume

Reply:
done.